tailwind.config.js

```diff
@@ -11,6 +11,7 @@ export default {
 			inherit: 'inherit',
 			transparent: 'transparent',
 			current: 'currentColor',
+			black: 'rgb(0, 0, 0)',
```

Comment:
You need this "color", using `color: initial` would not work out?

Reply:
Not against adding it nor strong opinion, really just asking.

Reply:
Unfortunately, `initial` doesn't work.

Reply:
🥲